src/main/java/frc/robot/commands/ClimbSequence1.java

```diff
@@ -23,14 +24,17 @@ public ClimbSequence1(ClimbSubsystem s) {
   public void initialize() {
     climbSubsystem.setAutoBoolean(true);
     climbSubsystem.deployPoles();
-    climbSubsystem.deployHooks();
+    climbSubsystem.unlockHooks();
   }
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
+    if(i>5){
     climbSubsystem.deployArms();
+    }
     climbSubsystem.resetClimbHeights();
+    i++;
```

Comment:
Unbounded increment isn't super great, but it won't overflow for many hours so it's fine for now.